pkg/agent/agent.go

```diff
@@ -87,6 +92,23 @@ func (a *Agent) readPump() {
 		return a.conn.SetReadDeadline(time.Now().Add(pongWait))
 	})
 
+	a.conn.SetCloseHandler(func(err int, msg string) error {
+		a.logger.Debug("peer disconnected", logging.Fields{"room": a.room, "peer": a.peer.UID, "error_code": err, "error_msg": msg})
```

Comment:
I don't think that the code passes to this handler is error code. It's just a code, so err_code might be misleading, since it's not necessarily an error.